### PR TITLE
fix(sidebar): padding on item properties versions

### PR DESCRIPTION
**Pull request by priyajeet**
Screen Shot 2019-06-11 at 4 21 23 PM

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @priyajeet has signed the CLA. Thanks for the pull request!

File: src/styles/variables.json [hunk 1]
```diff
@@ -89,5 +89,6 @@
     "overlay-z-index": "190",
     "tooltip-z-index": "190",
     "bdl-line-height": "20px",
-    "bdl-border-radius-size": "4px"
```
Comment (priyajeet) [inline src/styles/variables.json]: Yeah need to add variable building to prepush.sh

File: src/elements/content-sidebar/DetailsSidebar.scss [hunk 2]
```diff
@@ -13,12 +13,14 @@
...
-    }
+.bcs-DetailsSidebar-notices {
+    @include bcs-SidebarSection;
+}
 
-    .bcs-details-content:empty {
-        margin: 0;
-    }
+.bcs-DetailsSidebar-versions {
+    @include bcs-SidebarSection;
```
Comment (jstoffan) [inline src/elements/content-sidebar/DetailsSidebar.scss]: Seems like this probably shouldn't be a sidebar section at all? Looks like it just needs some bottom padding.

File: src/styles/variables.js [hunk 3]
```diff
@@ -92,3 +92,4 @@ export const overlayZIndex = '190'; // overlay-z-index
 export const tooltipZIndex = '190'; // tooltip-z-index
 export const bdlLineHeight = '20px'; // bdl-line-height
 export const bdlBorderRadiusSize = '4px'; // bdl-border-radius-size
+export const bdlTransitionDuration200Ms = '0.2s'; // bdl-transition-duration-200ms
```
Comment (jstoffan) [inline src/styles/variables.js]: Do we want to go with @booc0mtaco's suggestion of `bdlTransitionDuration-short`?

Comment (priyajeet) [reply src/styles/variables.js]: Another PR for that, this is auto-generated from prior PR.

**Pull request merged**